ADAL/src/urlprotocol/ADURLProtocol.m

```diff
@@ -145,14 +155,31 @@ + (BOOL)canInitWithRequest:(NSURLRequest *)request
         }
     }
     
-    AD_LOG_VERBOSE(_reqContext(request).correlationId, @"%@ - host: %@", @"+[ADURLProtocol canInitWithRequest:] ignoring handling of host", [request.URL host]);
+    if ([ADAuthorityUtils isKnownHost:request.URL])
+    {
+        AD_LOG_VERBOSE(_reqContext(request).correlationId, @"%@ - host: %@", @"+[ADURLProtocol canInitWithRequest:] ignoring handling of host", [request.URL host]);
```

Comment:
nit: why @"+[ADURLProtocol canInitWithRequest:] ignoring handling of host" needs to be passed as an argument?

Reply:
fixed